blackbox/docs/configuration.txt

```diff
@@ -694,42 +694,59 @@ Collecting Stats
   | *Default:*   ``10000``
   | *Runtime:*  ``yes``
 
-  The number of jobs kept in the ``sys.jobs_log`` table on each node
-  for performance analytics. Older entries will be deleted when the
-  ``jobs_log_size`` is reached. A single SQL statement results in a
-  job to be executed on the cluster. A higher number results in more
-  expressive results but also in more occupied RAM. Setting it to
-  ``0`` disables collecting job information.
+  The number of jobs kept in the :ref:`sys.jobs_log <sys-logs>` table on each
+  node for performance analytics. Older entries will be deleted when the size is
+  reached. A single SQL statement results in a job to be executed on the
+  cluster. A higher number results in more expressive results but also in more
+  occupied RAM. Setting it to ``0`` disables collecting job information.
 
 .. _stats.jobs_log_expiration:
 
 **stats.jobs_log_expiration**
   | *Default:*  ``0s`` (disabled)
   | *Runtime:*  ``yes``
 
-  This setting specifies the period when jobs are stored in the ``sys.jobs_log``
-  table. It periodically clears entries that are completed and older than the
-  specified time interval. This setting overrides ``jobs_log_size`` and logs a
-  warning message if both settings are set explicitly. If the value is set to
-  ``0`` the setting gets disabled.
+  This setting specifies the time period when jobs are cleared out of the
```

Comment:
Entries are cleared by a periodically triggered job if they are older than the specified expire time.